frontend/src/components/TopBar/TopBar.js

```diff
@@ -61,7 +57,7 @@ export default function TopBar({
         elevation={1}
       >
         <Toolbar>
-          <Grid container alignItems="center" spacing={1}>
+          <Grid container direction="row" alignItems="center">
```

Comment:
direction="row" is not needed as it is the default value of Grid component

Reply:
I like the explicit way